corehq/apps/repeaters/models.py

```diff
@@ -132,6 +133,7 @@ class Repeater(QuickCachedDocumentMixin, Document, UnicodeMixIn):
     format = StringProperty()
 
     use_basic_auth = BooleanProperty(default=False)
+    auth_type = StringProperty(choices=("basic", "digest", ""))
```

Comment:
does empty string have a special significance? note that even though the choices are passed here, couchdbkit will still allow a value of None (and also default the value to None if it's not specified)